### PR TITLE
1657393: Add missing descriptions

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
## Summary

We are adding descriptions to improve SEO, per [1657393](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1657393). Suggestions are welcome, but keep in mind our 100-160 char range. 

@KrzysztofCwalina - You're listed as author for these topics. Please review when you get a chance. Our SLA specifies 5 business days for review, but let me know if you need more time. (Also, let me know if you are not the right person to approve. We go by the github author tag.)

Fixes #1657393

### CODE REVIEW

File: docs/standard/design-guidelines/names-of-classes-structs-and-interfaces.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Names of Classes, Structs, and Interfaces"
+description: Use these guidelines for naming Classes, Structs, and Interfaces as part of guidelines for designing libraries that extend and interact with the .NET Framework.
```
Comment (BillWagner) [inline docs/standard/design-guidelines/names-of-classes-structs-and-interfaces.md:3]: small nit: These guidelines apply to .NET Core as well.

```suggestion
description: Use these guidelines for naming Classes, Structs, and Interfaces as part of guidelines for designing libraries that extend and interact .NET libraries.
```

File: docs/standard/design-guidelines/names-of-namespaces.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Names of Namespaces"
+description: Use these guidelines for naming namespaces as part of guidelines for designing libraries that extend and interact with the .NET Framework.
```
Comment (BillWagner) [inline docs/standard/design-guidelines/names-of-namespaces.md:3]: Same suggestion here:

```suggestion
description: Use these guidelines for naming namespaces as part of guidelines for designing libraries that extend and interact with .NET libraries.
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks @TimShererWithAquent 

I had a couple small nits to change because these articles would apply equally well to .NET Core.

Once you accept those changes, this is ready to merge.

**Comment by TimShererWithAquent** [comment]
@BillWagner Thanks - your changes are accepted.

**Review by IEvangelist — APPROVED** [review]
Looks good to me :shipit:

**Comment by TimShererWithAquent** [comment]
Going to close and reopen.

**Comment by TimShererWithAquent** [comment]
and reopen

**Comment by TimShererWithAquent** [comment]
These changes were accepted. Not sure why it still says blocked.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
@TimShererWithAquent Thanks for making the updates. I've approved it.

(Since I requested changes, I have to mark it as approved before it can be merged).

**Pull request merged**